src/NewCommand.php

```diff
@@ -208,9 +208,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 $this->configureDefaultDatabaseConnection($directory, $database, $name);
 
                 if ($migrate) {
-                    $this->runCommands([
-                        $this->phpBinary().' artisan migrate',
-                    ], $input, $output, workingPath: $directory);
+                    $commands = [
+                        trim(sprintf(
+                            $this->phpBinary().' artisan migrate %s',
+                            $database === 'sqlite' && ! $input->isInteractive() ? '--force' : '',
```

Comment:
Wouldn't this also can happen on MySQL database as well?

Reply:
Or, shouldn't it be executed with `--no-interaction` as well

Reply:
> Wouldn't this also can happen on MySQL database as well?

Ah yeah probably, I will update.

> Or, shouldn't it be executed with --no-interaction as well

I thought so, but `--no-interaction` prevents the database being created, see https://github.com/laravel/framework/blob/11.x/src/Illuminate/Database/Console/Migrations/MigrateCommand.php#L200, which just results in an error when the migrations then run. Not exactly sure why the `migrate` command works like that but it does.

Reply:
I believe `--no-interaction` behaviour on migrate should be respected here if you initiate laravel new without interaction.

Reply:
OK so call `php artisan --force --no-interaction` yeah?

Reply:
I wouldn't recommend `--force` unless `new` command also has the option, this would make it hard to maintain if in future `--force` contains action that not inline with `--no-interaction`.

Reply:
OK but if you just do `php artisan migrate --no-interaction` the migrate command will throw an exception:

> Illuminate\Database\QueryException
> Database file at path [/.../database/database.sqlite] does not exist.

Reply:
Why don't just touch the file within this command instead? Would that make more sense?

Reply:
Yeah can do. That would work.